.github/workflows/universalnrp-test-run.yml

```diff
@@ -21,6 +21,10 @@ on:
         description: 'Pester tag to include in the test run. If not provided, all tests will be run.'
         default: ''
         type: string
+      ref:
```

Comment:
Same as above, this should have the same implicit `ref` (context) as the workflow that calls this. Why do we need to explicitly specify a `ref`?

Reply:
i thought you had to be explicit, just read on [Contexts](https://docs.github.com/en/actions/learn-github-actions/contexts) and your right, good catch thanks!